controllers/customer_controller_test.go

```diff
@@ -39,14 +43,17 @@ var _ = Describe("Customer Controller", func() {
 					project := util.GetProject(mockData.CustomerTestProjectInput.Spec.Name, ns)
 					Expect(project.Status.ID).ToNot(Equal(""))
 
+					// Randomize customer name and email
+					mockData.SampleCustomer = cUtil.RandCustomer(mockData.SampleCustomer)
```

Comment:
Same as above. This should be outside. In the current scenario, the aftereach block won't be able to handle it